src/strings/en-us.json

```diff
@@ -373,6 +373,7 @@
     "HeaderFetchImages": "Fetch Images",
     "HeaderForKids": "For Kids",
     "HeaderFrequentlyPlayed": "Frequently Played",
+    "HeaderGuestCast": "Guest Cast",
```

Comment:
I suspect you went with "Guest Cast" here because the other section is "Cast & Crew", but that section needs overhauled, so let's just go with "Guest Stars"